src/command_modules/azure-cli-billing/azure/cli/command_modules/billing/commands.py

```diff
@@ -5,21 +5,30 @@
 
 # pylint: disable=line-too-long
 
-from azure.cli.core.commands import cli_command
-from azure.cli.core.profiles import supported_api_version, PROFILE_TYPE
-from azure.cli.command_modules.billing._client_factory import \
-    (invoices_mgmt_client_factory,
-     billing_periods_mgmt_client_factory)
+from azure.cli.core.sdk.util import CliCommandType
+from azure.cli.command_modules.billing._client_factory import (
+    invoices_mgmt_client_factory, billing_periods_mgmt_client_factory)
 from ._exception_handler import billing_exception_handler
 
-if not supported_api_version(PROFILE_TYPE, max_api='2017-03-09-profile'):
-    billing_periods_path = 'azure.mgmt.billing.operations.billing_periods_operations#'
-    custom_path = 'azure.cli.command_modules.billing.custom#'
 
-    def billing_command(*args, **kwargs):
-        cli_command(*args, exception_handler=billing_exception_handler, **kwargs)
+def load_command_table(self, _):
+    billing_invoice_util = CliCommandType(
+        min_api='latest',
```

Comment:
Please remove this.  You should put a "min_profile" kwarg in the init for the Commands Loader to handle this.